webdav.go

```diff
@@ -30,6 +30,10 @@ type WebDAV struct {
 	// Accepts placeholders.
 	Root string `json:"root,omitempty"`
 
+	// The prefix to prepend to paths.
+	// Accepts placeholders.
+	Prefix string `json:"prefix,omitempty"`
```

Comment:
Can we improve the godoc on this property? It's a little ambiguous/vague about what it actually does. I thought it stripped a path, not added one...

Reply:
Yeap should be more clear. Maybe some other term i better also? BasePath? RequestPath? 😕

Reply:
`PathPrefix` maybe? It's hard to decide without knowing really well what it actually does.

Reply:
Added more detailed description and example that i hope i more clear. Decided to keep the option name. Think it looks strange if we have `root` but then `path_prefix`, should root be `root_path` then?